packages/dbadmin/src/tools/init-schema-with-history.ts

```diff
@@ -16,7 +17,7 @@ export function listSchemas(): string {
         FROM information_schema.schemata;`
 }
 
-export function initSchemaWithHistory(schema: string): string {
+export function initSchemaWithHistory(schema: string, lionWebVersion: string): string {
```

Comment:
Can `schema` be renamed to `schemaName`, as it's a bit confusing – to me, at least – now?